routes/api/stream/index.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express');
+
+const router = express.Router();
+
+router.get('/', (req, res) => {
```

Comment:
we'll add it later. I think linting might fail if there are unused vars, and this is a mock for now.